src/module/macros.js

```diff
@@ -15,7 +15,7 @@ export async function createOseMacro(data, slot) {
     return ui.notifications.warn(
       game.i18.localize("OSE.warn.macrosOnlyForOwnedItems")
     );
-  const item = data.data;
+  const item = data;
```

Comment:
Why not just rename the param above to `item`?

Reply:
oops, turns out this is not an Item document object after all! So this code change breaks the (probably not often used) ability to drag an item from your sheet to your macro bar to create a roll macro (or if there is no roll for the item, a "view item" macro)

Reply:
This change has been undone. I'd like to refactor this as part of a different PR